src/patterns/dosdp-dev/abnormalConstrictionOfAnatomicalEntity.yaml

```
@@ -1,17 +1,24 @@
 pattern_name: abnormalConstrictionOfAnatomicalEntity
 pattern_iri: http://purl.obolibrary.org/obo/upheno/patterns-dev/abnormalConstrictionOfAnatomicalEntity.yaml
-description: "An anatomical entity that is being drawn together, compressed or squeezed physically, such as HP_0001650	Aortic valve stenosis."
+description: "An anatomical entity that is being drawn together, compressed or squeezed physically."
+
+# examples
+# http://purl.obolibrary.org/obo/HP_0001650      # Aortic valve stenosis
+# http://purl.obolibrary.org/obo/HP_0002043      # Esophageal stricture
+# http://purl.obolibrary.org/obo/MP_0003389      # constriction of pericardium
 
 contributors:
-- https://orcid.org/0000-0001-5208-3432
+- https://orcid.org/0000-0001-5208-3432  # Nicole Vasilevsky
```

```suggestion
  - https://orcid.org/0000-0001-5208-3432  # Nicole Vasilevsky
```
You need to indent these as required by yaml syntax. You can check your yaml syntax with [yamllint](https://yamllint.readthedocs.io/en/stable/):
```
yamllint abnormalConstrictionOfAnatomicalEntity.yaml 
```